### PR TITLE
MES-4542 - Reversing diagram update

### PR DESCRIPTION
**Pull request by TomBillingtonUK**
## Description
- Updates reversing diagram for Cat C Sub Categories 

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by amobarak — APPROVED** [review]

**Review by faggc — APPROVED** [review]

**Review by MarkPatt89 — APPROVED** [review]

File: src/providers/reversing-distances/__tests__/reversing-distances.spec.ts [hunk 1]
```diff
@@ -18,18 +17,27 @@ describe('ReversingDistancesProvider', () => {
...
+      vehicleWidth: 2,
+    };
+
     describe('Category C', () => {
       it('should return a start distance value 3 and a half times the vehicle length', () => {
         const result = reversingDistancesProvider.getDistanceLength(vehicleDetails, TestCategory.C);
         expect(result.startDistance).toEqual(52.5);
       });
 
+      it('should return a value 3 and a half times vehicle length is greater then 16.5', () => {
```
Comment (mgoldson) [inline src/providers/reversing-distances/__tests__/reversing-distances.spec.ts:36]: This description doesn't quite read right...
Maybe 'should return a value 3 and a half times vehicle length **if** greater **than** 16.5' ?

File: src/providers/reversing-distances/__tests__/reversing-distances.spec.ts [hunk 2]
```diff
@@ -42,6 +50,11 @@ describe('ReversingDistancesProvider', () => {
         expect(result.startDistance).toEqual(60);
       });
 
+      it('should return a start distance of 66 if the vehicle length is greater then 16.5', () => {
```
Comment (mgoldson) [inline src/providers/reversing-distances/__tests__/reversing-distances.spec.ts:53]: than

File: src/providers/reversing-distances/__tests__/reversing-distances.spec.ts [hunk 3]
```diff
@@ -54,6 +67,11 @@ describe('ReversingDistancesProvider', () => {
         expect(result.startDistance).toEqual(52.5);
       });
 
+      it('should return a value 3 and a half times vehicle length is greater then 16.5', () => {
```
Comment (mgoldson) [inline src/providers/reversing-distances/__tests__/reversing-distances.spec.ts:70]: This description doesn't quite read right...
Maybe 'should return a value 3 and a half times vehicle length **if** greater **than** 16.5' ?

File: src/providers/reversing-distances/__tests__/reversing-distances.spec.ts [hunk 4]
```diff
@@ -66,6 +84,11 @@ describe('ReversingDistancesProvider', () => {
         expect(result.startDistance).toEqual(60);
       });
 
+      it('should return a start distance of 66 if the vehicle length is greater then 16.5', () => {
```
Comment (mgoldson) [inline src/providers/reversing-distances/__tests__/reversing-distances.spec.ts:87]: than

File: src/providers/reversing-distances/reversing-distances.ts [hunk 5]
```diff
@@ -1,8 +1,16 @@
...
 import { ReversingLengths } from './reversing-lengths.model';
+import { CatCUniqueTypes } from '@dvsa/mes-test-schema/categories/C';
+import { CatC1UniqueTypes } from '@dvsa/mes-test-schema/categories/C1';
+import { CatCEUniqueTypes } from '@dvsa/mes-test-schema/categories/CE';
+
+export type CatCVehicleDetailsUnion =
+  | CatCUniqueTypes.VehicleDetails
+  | CatC1UniqueTypes.VehicleDetails
+  | CatCEUniqueTypes.VehicleDetails
+  | CatC1UniqueTypes.VehicleDetails;
```
Comment (mgoldson) [inline src/providers/reversing-distances/reversing-distances.ts:13]: Should this be C1E unique types?

**Review by mgoldson — COMMENTED** [review]
Nice work! A few questions worth looking at...

**Pull request merged**